### ISSUE TITLE
How to get per pixel logits?

### ISSUE THREAD

**Issue by AceFromSpace** [open-mmlab/mmsegmentation#1278]
Hi 

I am working on model on  segmentation model . Right now It works and I am getting masks for each classes but I need logit ("probabilities") for each pixel  not only labels.

How to obtain that ?

**Comment by MengzhangLI** [comment]
https://github.com/open-mmlab/mmsegmentation/blob/a39f5856ce514ca09a161119041fb8490354c18f/mmseg/models/segmentors/encoder_decoder.py#L256

**Comment by AceFromSpace** [comment]
Ok thanks. But Is there an cleaner way to obtain logits? 
I mean config modification without source code changes.

**Comment by MengzhangLI** [comment]
Not supported yet.

But we would add metric which may use logits (such as AUC) this quarter.
